features/list_dishes_on_landing_page.feature

```diff
@@ -0,0 +1,11 @@
+Feature: List dishes on landing page
+  As a customer/visitor
+  In order to decide on what dish to order
+  I would like to see a list of available dishes the moment the site displays
+
+  Scenario: Visitor see a list of dishes on the page
+    Given there is a dish called "Hot Dog" with a price of "20" with description of "Great Hot Dog" and category of "Starter" in the system
+    And there is a dish called "Hamburger" with a price of "30" with description of "Great Hamburger" and category of "Main" in the system
+    And there is a dish called "Grilled Cheese" with a price of "50" with description of "Great Grilled Cheese" and category of "Dessert" in the system
+    When I visit the site
+    Then I would like to see "Hot Dog" with the price of "20 kr"
```

Comment:
What else?  I see that you are grouping the products in categories. Please reflect that in this test (modify the title) and/or create another scenario. Please involve your teammates in this work. We would like to see every other commit made by another developer.

Reply:
Alright made a new commit with the updated Scenario which includes the fact that the dishes are grouped by category, Lisa and I will make sure to have interchanging commits when we start on the order feature. Not sure if interchanging is the correct term, but I think you know what I mean :P